### ISSUE TITLE
Create the Bottom Bar from XAML

### ISSUE THREAD

**Issue by nosmirck** [thrive-now/BottomNavigationBarXF#39]
I've managed to install the package and made it work with my project using the example code, but, we are trying to have zero code-behind and we define everything using the xaml page.

When I create a new xaml page and inherit from BottomBarPage (also changing the respective code on the XAML part to BottomBarPage) it fails on compilation because the generated compiled xaml to cs tries to look for BottomBarPage inside Xamarin.Forms.

What can I do? Is there any example on how to set up this bottom bar page with xaml?

Thanks in advance for any help.

**Comment by jjTudu** [comment]
same problem here

**Comment by mhdshbk** [comment]
@nosmirck  Have you found any way to setup this bottom bar page with XAML?

**Comment by nosmirck** [comment]
@mhdskbk no, there's no way to do it from xaml for now I guess. The issue is still open and the developer has not answered anything.

**Comment by jjTudu** [comment]
i think we have a solution now, [bottom-navigation-bar](https://blog.xamarin.com/exploring-androids-bottom-navigation-view/). have not tried yet
